pages/api/maps/flights3D.tsx

```diff
@@ -27,7 +27,7 @@ export default async function handler(
     deviceScaleFactor: 6,
   });
 
-  await page.goto(`${process.env.SITE_URL}/flights2019/flights3D`);
+  await page.goto(`http://localhost:3000/flights2019/flights3D`);
```

Comment:
I would arguing against keeping the page `flights3D.tsx` and rather change the url (please check before accepting):
```suggestion
  await page.goto(`http://localhost:3000/travels2019`);
```

Reply:
I can removed this file, but i am not sure do we still need the flights2019 folder and index.tsx file?